io/disk/lvsetup.py

```diff
@@ -63,21 +63,24 @@ def setUp(self):
                 pkg = 'btrfs-progs'
         if pkg and not smm.check_installed(pkg) and not smm.install(pkg):
             self.cancel("Package %s could not be installed" % pkg)
+
         lv_snapshot_name = self.params.get(
             'lv_snapshot_name', default='avocado_sn')
         self.ramdisk_basedir = self.params.get(
             'ramdisk_basedir', default=os.path.join(self.workdir, 'ramdisk'))
         self.ramdisk_sparse_filename = self.params.get(
             'ramdisk_sparse_filename', default='virtual_hdd')
 
-        if lv_utils.vg_check(vg_name):
-            self.cancel('Volume group %s already exists' % vg_name)
         self.vg_name = vg_name
-        if lv_utils.lv_check(vg_name, lv_name):
-            self.cancel('Logical Volume %s already exists' % lv_name)
         self.lv_name = lv_name
```

Comment:
same here.

Reply:
@Naresh-ibm can be done, but that is not in the scope of this commit. Can be taken later.